packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -158,7 +161,12 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     double? maxHeight,
     int? imageQuality,
   }) {
-    return getFiles(accept: _kAcceptImageMimeType, multiple: true);
```

Comment:
Same here, determine if resizing is needed, and then return imageResizer.resizeImages(getFiles(...), maxWidth, maxHeight, imageQuality);